runtime/kusama/src/lib.rs

```diff
@@ -564,6 +564,8 @@ parameter_types! {
 	pub const MaximumReasonLength: u32 = 16384;
 	pub const BountyCuratorDeposit: Permill = Permill::from_percent(50);
 	pub const BountyValueMinimum: Balance = 2 * DOLLARS;
+	pub const MaxActiveSubBountyCount: u32 = 10;
+	pub const MinimumCurationFee:u64 = 1;
```

Comment:
I don't think 1 is a good minimum, this is very very low. better to have `1 * DOLLARS`

Reply:
Sure it makes sense